src/composables/use-pages.js

```diff
@@ -25,9 +25,9 @@ export default function usePages() {
       link: app.localePath('/search-help'),
     },
     {
-      id: 'meta-search',
-      name: 'header.meta-search-nav-item',
-      link: app.localePath('/meta-search'),
+      id: 'additional-sources',
+      name: 'header.additional-sources-nav-item',
+      link: app.localePath('/additional-sources'),
```

Comment:
Since you've changed the link, you will need to rename `src/pages/meta-search.vue` to `src/pages/additional-sources.vue`. Nuxt uses file names to map to URLs.

Reply:
Do we also want to map a redirect for this link?